src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -29,7 +35,9 @@ class MockWorkflowManagerActor extends Actor  {
 
   def receive = {
     case SubmitWorkflow(wdl, inputs) =>
-      println("not yet implemented")
+      Future {
+            MockWorkflowManagerActor.submittedWorkflowId
+      }.pipeTo(sender())
```

Comment:
Why is this creating a `Future` around the ID to then do a `pipeTo`?  Why not just  
```sender ! MockWorkflowManagerActor.submittedWorkflowId```